exports/ystdlib-py/src/ystdlib/find.py

```diff
@@ -33,6 +33,8 @@ def find(
     indicates all filenames should be included.
     :return: Matched paths.
```

Comment:
Unrelated to the PR, but this function's docstring is missing `root_paths`. Could add it in this PR if you like.